src/RowRenderer.tsx

```diff
@@ -63,41 +57,26 @@ function RowRenderer<R, K extends keyof R>({
     enableCellRangeSelection: props.enableCellRangeSelection
   };
 
-  function renderGroupRow() {
-    const { ref, columns, ...rowGroupProps } = rendererProps;
-
-    return (
-      <RowGroup<R>
-        {...rowGroupProps}
-        {...__metaData!}
-        columns={columns}
-        name={(rowData as RowData).name!}
-        eventBus={eventBus}
-        renderer={rowGroupRenderer}
-        onRowExpandToggle={props.onRowExpandToggle}
-        renderBaseRow={(p: IRowRendererProps<R>) => <Row ref={ref} {...p} />}
-      />
-    );
-  }
-
   if (__metaData) {
     if (__metaData.getRowRenderer) {
       return __metaData.getRowRenderer(rendererProps, idx);
     }
     if (__metaData.isGroup) {
-      return renderGroupRow();
+      return (
+        <RowGroup<R>
+          {...rendererProps}
+          {...__metaData!}
+          name={(rowData as RowData).name!}
+          eventBus={eventBus}
+          renderer={rowGroupRenderer}
+          onRowExpandToggle={props.onRowExpandToggle}
+          renderBaseRow={(p: IRowRendererProps<R>) => <Row {...p} />}
```

Comment:
I think we should remove `renderBaseRow`. IIRC it was added to forward the ref correctly to prevent breaking `setScrollRef` logic.

Reply:
We can handle it it a separate PR if you prefer